components/Leaderboard/Leaderboard.tsx

```diff
@@ -223,13 +225,14 @@ export const DisplayThirdPerson: React.FC<PersonPlace> = ({ data, place }) => {
             <span>{data.stats.commits}</span>
           </div>
         </div>
-        <div>
-          <h4 className='text-xs '>Projects:</h4>
+        <div className="flex gap-2 flex-wrap">
+          <h4 className="text-xs flex-grow w-full">Projects:</h4>
           {data.projects_names.map((project, ind) => {
             return (
               <a
                 key={ind}
-                className="flex gap-2 items-center underline text-indigo-300 text-xs"
+                href={`https://github.com/${project.url}`}
+                className="flex gap-1 items-center underline text-indigo-300 text-xs"
```

Comment:
@barel-mishal 
Looks good please fix that review item asap so we can merge this in

Reply:
I have saved it as a variable. Please note that this design is subject to change. This is why I did not prioritize making the code look better. While I believe it is very important to have clean code, due to the rapid pace of changes in the design and interface, I think it would be better to address this in a separate issue.